app/src/main/java/com/zulip/android/activities/ZulipActivity.java

```diff
@@ -328,6 +329,10 @@ public void onClick(View v) {
             }
         });
         etSearchStream = (EditText) findViewById(R.id.stream_drawer_search);
+        if (isCurrentThemeNight) {
+            etSearchPeople.setTextColor(ContextCompat.getColor(this, R.color.colorPrimary));
```

Comment:
@Sam1301 Color assigned to editText by default is white only (in black theme).But white Color was kind of invisible because of white Color of background of editText.

Yes it may cause some issue in near future if we change colourPrimary. To avoid this we can add new value in colors.xml.
@kunall17 What do you think about this?